### PR TITLE
Add sets to cwrapper

### PR DESCRIPTION
**Pull request by rikardn**

### CODE REVIEW

**Comment by rikardn** [comment]
This is ready for review

File: symengine/cwrapper.cpp [hunk 1]
```diff
@@ -658,6 +662,164 @@ void basic_str_free(char *s)
     delete[] s;
 }
 
+void bool_true(basic s)
```
Comment (isuruf) [inline symengine/cwrapper.cpp:665]: `basic_set_bool_true`?

Comment (isuruf) [reply symengine/cwrapper.cpp:665]: Or `basic_bool_set_true`?

Comment (rikardn) [reply symengine/cwrapper.cpp]: Ok, we have `integer_set_si` so I changed to `bool_set_true` and `bool_set_false`

**Comment by rikardn** [comment]
Thanks @Marlin-Na for reviewing. I agree. Perhaps even better would be to have a symengine prefix then class and then function. Something like `symengine_set_is_subset` or `se_set_is_subset`, but this would go against the current naming scheme. Otherwise I think `set_is_subset` is the closest to the current naming scheme. What do you think?

**Comment by rikardn** [comment]
Can this be merged?

**Comment by rikardn** [comment]
Ok. Is this better?

**Review by isuruf — APPROVED** [review]

**Comment by rikardn** [comment]
Thanks @isuruf and @Marlin-Na for reviewing

**Pull request merged**